<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkBoxAverage">
           <property name="text">
            <string>Visualize average</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBoxAverage"/>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBoxStd"/>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="checkBoxStd">
           <property name="text">
            <string>Visualize standard deviation</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelSubject">
           <property name="text">
            <string>Visualize data as</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="comboBoxSubject"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <widget class="QLabel" name="labelSize">
           <property name="text">
            <string>Size</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Colormap</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBoxColormap"/>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="spinBoxMin"/>
         </item>
         <item row="1" column="2">
          <widget class="QSpinBox" name="spinBoxMax"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
